README.md

```diff
@@ -41,6 +42,13 @@ To create a share, the sending server SHOULD make a HTTP POST request to the `/s
 ### Share Acceptance
 In response to a share creation, the receiving server MAY send back a [notification](https://cs3org.github.io/OCM-API/docs.html?branch=develop&repo=OCM-API&user=cs3org#/paths/~1notifications/post) to the sending server, with  `notificationType` set to `"SHARE_ACCEPTED"` or `"SHARE_DECLINED"`. The sending server MAY expose this information to the end user. 
 
+### Share Access
+To access a share, the receiving server MAY use multiple ways, depending on the received payload and on the `protocol.name` property:
+
+* If `protocol.name` = `multi`, the receiver SHOULD make a HTTP PROPFIND request to `protocol.webdav.uri` to access the remote resource. If `protocol.webdav.sharedSecret` is not empty, the receiver SHOULD pass it as a `Authorization: bearer` header.
+
+* If `protocol.name` = `webdav`, the receiver SHOULD inspect the `protocol.options` property. If it contains a `sharedSecret`, as in the [legacy example](https://cs3org.github.io/OCM-API/docs.html?branch=develop&repo=OCM-API&user=cs3org#/paths/~1shares/post), then the receiver SHOULD make a HTTP PROPFIND request to `https://<token>:@<host><path>/<rel_path>`, where: `<host>` is the remote server, `<path>` is obtained by querying the [Discovery](#discovery) endpoint and extracting `resourceTypes[0].protocols.webdav`, and optionally `<rel_path>` is the relative path of the resource within the share, when needed.
```

Comment:
Counter-proposal from @yasharpm in today's meeting:
```
"protocol":{
  "name":"webdav",
  "options": {"sharedSecret"}
  "webdav":{ "sharedSecret" },
  "webapp":{...}
}
```
I like @glpatcern's proposal better